api/imapsync/read

```diff
@@ -146,14 +141,13 @@ if ($cmd eq 'list') {
         $crypt = '';
     }
 
-    # add double-quotes to password for Mail::IMAPClient bug
-    my @imapsync =("imapsync","--host2","localhost","--port2","143","--tls2","--user2",
+    my @imapsync =("/usr/bin/imapsync","--host2","localhost","--port2","143","--tls2","--user2",
         $name.'*vmail',"--passfile2","/var/lib/nethserver/imapsync/vmail.pwd","--host1",$hostname,
-        "--port1",$Port,$crypt,"--user1",$username,"--password1",'"'.$password.'"',
+        "--port1",$Port,$crypt,"--user1",$username,"--passfile1","/var/lib/nethserver/imapsync/".$name.".pwd",
         "--exclude","'^Public Folders$|^Trash$|^Deleted Items$|^Public".$exclude."'",
```

Comment:
:warning: `$|` is a special Perl variable https://perldoc.perl.org/perlvar#$%7C

Furthermore, as Bash is not invoked by open() any more, the single quotes should be removed

```suggestion
        "--exclude", q(^Public Folders$|^Trash$|^Deleted Items$|^Public) . $exclude,
```